mobile/src/stateMachine/views/strategies/QolHistoryStrategiesView.tsx

```diff
@@ -24,6 +24,7 @@ export class QolHistoryStrategiesView extends ReviewStrategiesView {
   }
 
   renderContent() {
+    console.log('this.strategyViewModel.temporaryDisplay', this.strategyViewModel.temporaryDisplay)
```

Comment:
Is this console log needed? Do we need to put a TODO comment to remove it later?